src/styles/block/page.scss

```diff
@@ -0,0 +1,5 @@
+.page {
+  box-sizing: border-box;
```

Comment:
it would be better if you will write  * {
                                                          box-sizing: border-box;
                                                       }
In this case, you don't duplicate your code and can get rid of 2px your in @each